atd-toolbox/data_model/python/sql_templates/fatalities_view.sql

```diff
@@ -0,0 +1,47 @@
+drop table fatalities cascade;
```

Comment:
I was in a tangled mess of migrations and so I downloaded a fresh copy of prod and did the up migrations, that worked fine, but when I went back down and tried to apply migrations again, I got an error here

`"message": "table \"fatalities\" does not exist",`

so I changed this to `drop table if exists fatalities cascade;`

i probably messed something up in my migrations snafu, but thought id mention it anyway.

Reply:
@roseeichelmann would you mind making that change to the migration like Chia did? would be nice to handle this edge case